docs/caph/01-getting-started/01-introduction.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: Introduction
+metatitle: "Cluster API for Hetzner: Compatibility and Quickstart Guide"
+sidebar: Introduction
+description: Hetzner's Cluster API makes setting up Kubernetes clusters easy on cloud or bare-metal. Compatible with CAPI v1.7.x, v1.8.x, and various Kubernetes versions.
```

Comment:
"Hetzner's Cluster API". Hetzner does not have a Cluster API. We don't provide this API.